From b436821ef3ba0f98ab7cf3c1d3aee2db79aba38f Mon Sep 17 00:00:00 2001
From: Jason Wang <jasowang@redhat.com>
Date: Thu, 9 Oct 2014 06:13:28 -0400
Subject: [virt] hyperv: fcopy_open() can be static

Message-id: <1412835210-22435-10-git-send-email-jasowang@redhat.com>
Patchwork-id: 97291
O-Subject: [RHEL7.1 PATCH V2 09/11] Drivers: hv: fcopy_open() can be static
Bugzilla: 1146358
RH-Acked-by: Vitaly Kuznetsov <vkuznets@redhat.com>

From: Fengguang Wu <fengguang.wu@intel.com>

Bugzilla: 1146358

CC: "K. Y. Srinivasan" <kys@microsoft.com>
CC: "Greg Kroah-Hartman" <gregkh@linuxfoundation.org>
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
(cherry picked from commit c765a6dfad10b857e9681d7c4dec375ebe78db8a)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/hv/hv_fcopy.c b/drivers/hv/hv_fcopy.c
index 285b5e2..eaaa3d8 100644
--- a/drivers/hv/hv_fcopy.c
+++ b/drivers/hv/hv_fcopy.c
@@ -327,7 +327,7 @@ static ssize_t fcopy_write(struct file *file, const char __user *buf,
  return sizeof(int);
 }
 
-int fcopy_open(struct inode *inode, struct file *f)
+static int fcopy_open(struct inode *inode, struct file *f)
 {
  /*
   * The user level daemon that will open this device is
@@ -344,7 +344,7 @@ int fcopy_open(struct inode *inode, struct file *f)
  return 0;
 }
 
-int fcopy_release(struct inode *inode, struct file *f)
+static int fcopy_release(struct inode *inode, struct file *f)
 {
  /*
   * The daemon has exited; reset the state.
-- 
1.7.1